utils/make-sqlite.py

```diff
@@ -37,7 +37,12 @@
 c.execute('''DROP TABLE IF EXISTS targets''')
 c.execute('''CREATE TABLE targets
              (host TEXT,
+              platform_id INTEGER,
               ruleset_id INTEGER)''')
+c.execute('''DROP TABLE IF EXISTS platforms''')
+c.execute('''CREATE TABLE platforms
```

Comment:
I was afraid you might say that :)
I figured I would just refactor this when the time came to get rid of `sqlite`, but I agree it adds complexity to something we're depricating. Refactoring it now.